counter/src/main/java/edu/nextstep/camp/counter/CounterActivity.kt

```diff
@@ -23,11 +23,12 @@ class CounterActivity : AppCompatActivity() {
     }
 
     private fun observeEvent() {
-        viewModel.liveEvent.observe(this) {
-            it.consume()?.let { event ->
+        viewModel.event.observe(this) {
+            val event = it.consume()
+            if (event != null) {
```

Comment:
```suggestion
            val event = it.consume() ?: return@observe
```
return 문을 적절하게 활용하신다면, 불필요한 depth를 줄일 수 있어요.